test/runtests.jl

```diff
@@ -1,62 +1,64 @@
+Pkg.checkout("StaticArrays", "sd/fixedsizearrays")
```

Comment:
@SimonDanisch now that juliacon has passed, could we get FreeTypeAbstraction up to date?  there are 2 PRs and 2 issues that would be nice to close out.